nilearn/plotting/surf_plotting.py

```diff
@@ -446,7 +447,13 @@ def _plot_surf_matplotlib(coords, faces, surf_map=None, bg_map=None,
     limits = [coords.min(), coords.max()]
 
     # set view
-    elev, azim = _set_view_plot_surf_matplotlib(hemi, view)
+    if isinstance(view, str):
+        elev, azim = _set_view_plot_surf_matplotlib(hemi, view)
+    elif isinstance(view, collections.Sequence) and len(view) == 2:
```

Comment:
`collections.Sequence` is deprecated since Python 3.3 and was scheduled to be removed in Python 3.10, which is why the tests with Python 3.10 are currently failing. You need to import `Sequence` from `collections.abc`:

```python
from collections.abc import Sequence
```

Reply:
I think this work (including raising the exception) should be done in `_set_view_plot_surf_matplotlib`. (btw its name probably shouldn't be `set...` as it doesn't modify any object but returns values it computed

Reply:
also this should use the `_check_view_is_valid` helper instead of duplicating its logic